prow/config/config.go

```diff
@@ -176,7 +176,30 @@ type Plank struct {
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
 	// JobURLPrefix is the host and path prefix under
 	// which job details will be viewable
+	// It should never be accessed directly, use `GetJobURLPrefix` instead
 	JobURLPrefix string `json:"job_url_prefix,omitempty"`
```

Comment:
I really hate that this is public, but unfortunately it is required for the YAML parsing of the config